futures/src/lib.rs

```diff
@@ -238,6 +238,8 @@ pub mod prelude {
         Poll,
     };
 
+    pub use futures_core::task::Context as TaskContext;
```

Comment:
@cramertj I think you followed the RFC more closely than I, but is this what the RFC concluded?

Reply:
When the RFC concluded, I said we'd leave it as `task::Context`, and that further prelude-import-renamings could be addressed via followup discussions/proposals. I'd be inclined to leave this out for now, and instead import `task`.

Reply:
That's fine. It's an easy change.